### PR TITLE
Update / clean up deps

### PR DESCRIPTION
**Pull request by ForsakenHarmony**

### CODE REVIEW

File: packages/cli/lib/fs.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+const { readFile, writeFile, copyFile, stat, statSync } = require('fs');
+const { promisify } = require('util');
+
+module.exports = {
+	copyFile: promisify(copyFile),
+	readFile: promisify(readFile),
+	writeFile: promisify(writeFile),
+	stat: promisify(stat),
+	statSync,
+};
```
Comment (ForsakenHarmony) [inline packages/cli/lib/fs.js]: was kinda annoyed by the abandoned dependency and we'll get native promise support 🔜™

**Comment by ForsakenHarmony** [comment]
This is now waiting for https://github.com/terkelg/tiny-glob/issues/35

**Comment by lukeed** [comment]
Node 6 is still intentionally supported. Can't merge this without major bump.

**Comment by lukeed** [comment]
_Removed my comment...I was distracted, sorry. Thought I was looking at a PR for an entirely different repo_ 🙈

**Comment by reznord** [comment]
Need to change the lerna version in `lerna.json` as well.

File: packages/cli/package.json [hunk 2]
```diff
@@ -58,67 +64,57 @@
...
     "html-webpack-plugin": "^3.2.0",
-    "inquirer": "^5.2.0",
+    "inquirer": "^6.2.2",
     "ip": "^1.1.5",
     "isomorphic-unfetch": "^3.0.0",
+    "kleur": "^3.0.1",
     "loader-utils": "^1.1.0",
-    "log-symbols": "^2.1.0",
-    "mini-css-extract-plugin": "^0.4.0",
+    "loglevelnext": "^3.0.0",
```
Comment (lukeed) [inline packages/cli/package.json]: Where does this get used?

File: packages/cli/lib/commands/create.js [hunk 3]
```diff
@@ -1,12 +1,10 @@
...
-const glob = require('glob');
+const glob = require('tiny-glob');
 const gittar = require('gittar');
-const fs = require('fs.promised');
-const copy = require('ncp');
-const { green } = require('chalk');
+const fs = require('../fs');
+const { green } = require('kleur');
 const { resolve, join } = require('path');
 const { prompt } = require('inquirer');
```
Comment (lukeed) [inline packages/cli/lib/commands/create.js]: I would vote for `prompts` here – waaay smaller but up to you

File: packages/cli/package.json (hunk 2 above)
Comment (ForsakenHarmony) [reply packages/cli/package.json]: oh, accidentally added it, will remove it again

File: packages/cli/lib/commands/create.js (hunk 3 above)
Comment (ForsakenHarmony) [reply packages/cli/lib/commands/create.js]: I don't mind

**Comment by reznord** [comment]
Seems like https://github.com/terkelg/tiny-glob/pull/41 is fixed, so this sounds good to me

**Comment by reznord** [comment]
On the other hand, should we do a beta with ceviche?

**Comment by lukeed** [comment]
@reznord It's not fixed. I merged it to continue fixing it in another PR. One of its sub-dependencies was/is broken.

**Pull request merged**